package.json

```diff
@@ -112,5 +112,8 @@
     "babel-core": "^7.0.0-bridge.0",
     "styled-components": "^5.3.1",
     "typescript": "^4.6.3"
+  },
+  "dependencies": {
+    "react-table-scrollbar": "^0.1.11"
```

Comment:
Are you sure you need this? I don't see any usage of that library in the changes below

Reply:
not needed anymore, I use css style only

Reply:
Pls remove then. And from the yarn.lock as well

Reply:
yes, removed